lib/convert-twig-includes-paths.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var utils = require('./utils');
+
+
+
+/*
+ * Convert file paths in twig include statements
+ *
+ * @param {Object} options  pattern importer options
+ * @param {String} twigContent  content from a twig file
+ *
+ * @returns {String} twigContent twig file content with converted paths
+*/
+function convertTwigIncludes (options,twigContent) {
+
+  if(!options.convertCategoryTitles){
+    return twigContent;
+  }
+
+  // regex expression to find twig include code blocks
+  var twigIncludeRegex = /{%.*include.*%}/g;
```

Comment:
It's because multiline "m" flag is not set, in one line that regex doesn't quite work - http://regexr.com/3bel3. Anyway, "?" should be always appeared to match first occurrence and get expected result